bears/php/PHPCodeSnifferBear.py

```diff
@@ -17,7 +17,9 @@ class PHPCodeSnifferBear:
     """
 
     LANGUAGES = {'PHP', 'JavaScript', 'CSS'}
-    REQUIREMENTS = {DistributionRequirement(apt_get='php-codesniffer')}
+    REQUIREMENTS = {AnyOneOfRequirements([
```

Comment:
please indent these better, with indents for each bracket, and closing comma's *everywhere*, so that we can add more requirements by adding only a single line, not modifying your original lines.

```
    REQUIREMENTS = {
        AnyOneOfRequirements(
...
        ),
    }
```